File: slo.dart
#!/usr/bin/env dart
class Point {
  var x, y;
  Point(this.x, this.y);
}

main() {
  var total = 0;
  for (var i = 0; i < 1000000; i++) {
    Point p = Point(2, 3);
    total = total + p.x + p.y;
  }
  print(total);
}

File: slo.rb
#!/usr/bin/env ruby
class Point
  attr_reader :x, :y
  def initialize(x: 0, y: 0)
    @x = x
    @y = y
  end
end

total = 0
1000000.times do
  p = Point.new(x: 2, y: 3)
  total = total + p.x + p.y
end

puts total
